src/ViewerComponents/ViewUtility.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Generates the an ID.
+ *
+ * @returns {string}
+ */
+export function generateID() {
+    return Math.random().toString(36).substring(8);
```

Comment:
I'm not a fan of randomly generated IDs, can we not use a counter or something similar?

Reply:
You're right. I will create after PR a store where a counter is included. For now I increase the length